src/scripts/main.js

```diff
@@ -1 +1,13 @@
 'use strict';
+
+const a = document.querySelector('#largeImg');
+const img = document.querySelector('#thumbs');
+
+img.addEventListener('click', (eve) => {
+  eve.preventDefault();
+  eve.target.closest('.list-item__link');
+
+  const link = eve.target.closest('.list-item__link');
+
+  a.src = link.href;
```

Comment:
Before assigning `link.href` to `a.src`, you should check if `link` is not `null`. If `link` is `null`, accessing `href` will throw an error. Consider adding a conditional check to ensure `link` is valid.